scripts/index.js

```diff
@@ -0,0 +1,182 @@
+const asideButton = document.getElementById("aside-button");
+const asideWrapper = document.getElementById("aside-wrapper");
+headerItemsRightPartMenuButton.addEventListener('click', () => {
+  asideWrapper.style.visibility = 'visible';
+  asideWrapper.style.opacity = 1;
+});
+document.addEventListener('click', (event) => {
+  if (event.target.classList.contains('aside-wrapper') || event.target.closest('.aside__button')) {
+    asideWrapper.style.visibility = 'hidden';
+    asideWrapper.style.opacity = 0;
+  }
+});
+asideWrapper.addEventListener('click', (event) => {
+  if (event.target.closest('.aside__list-element-button-skills')) {
+    mainSkills.scrollIntoView({ behavior: 'smooth' });
+    asideWrapper.style.visibility = 'hidden';
+    asideWrapper.style.opacity = 0;
+  }
+  else if (event.target.closest('.aside__list-element-button-portfolio')) {
+    mainPortfolio.scrollIntoView({ behavior: 'smooth' });
+    asideWrapper.style.visibility = 'hidden';
+    asideWrapper.style.opacity = 0;
+  }
+  else if (event.target.closest('.aside__list-element-button-video')) {
+    mainVideo.scrollIntoView({ behavior: 'smooth' });
+    asideWrapper.style.visibility = 'hidden';
+    asideWrapper.style.opacity = 0;
+  }
+  else if (event.target.closest('.aside__list-element-button-price')) {
+    mainPrice.scrollIntoView({ behavior: 'smooth' });
+    asideWrapper.style.visibility = 'hidden';
+    asideWrapper.style.opacity = 0;
+  }
+  else if (event.target.closest('.aside__list-element-button-contacts')) {
+    mainFeedback.scrollIntoView({ behavior: 'smooth' });
+    asideWrapper.style.visibility = 'hidden';
+    asideWrapper.style.opacity = 0;
+  };
+});
+
+
+const mainPortfolioNavigation = document.getElementById("main-portfolio-navigation");
+const mainPortfolioNavigationButtonWinter = document.getElementById("main-portfolio-navigation-button-winter");
+const mainPortfolioNavigationButtonSpring = document.getElementById("main-portfolio-navigation-button-spring");
+const mainPortfolioNavigationButtonSummer = document.getElementById("main-portfolio-navigation-button-summer");
+const mainPortfolioNavigationButtonAutumn = document.getElementById("main-portfolio-navigation-button-autumn");
+const mainPortfolioImagesImage1 = document.getElementById("main-portfolio-images-image-1");
+const mainPortfolioImagesImage2 = document.getElementById("main-portfolio-images-image-2");
+const mainPortfolioImagesImage3 = document.getElementById("main-portfolio-images-image-3");
+const mainPortfolioImagesImage4 = document.getElementById("main-portfolio-images-image-4");
+const mainPortfolioImagesImage5 = document.getElementById("main-portfolio-images-image-5");
+const mainPortfolioImagesImage6 = document.getElementById("main-portfolio-images-image-6");
+mainPortfolioNavigation.addEventListener('click', (event) => {
+  if (event.target.closest('.main__portfolio-navigation-button-winter')) {
+    mainPortfolioNavigationButtonWinter.style.color = 'black';
```

Comment:
We should not use inline-styles straightaway. We should use classes instead when possible. For exmaple:
```
.some-class {
    color: 'black',
    background-color: '#bdae82'
}
```
And then:
` mainPortfolioNavigationButtonWinter.classsList.add('some-class');`